cypress/integration/catalog/categories.js

```diff
@@ -70,7 +73,7 @@ filterTests({ definedTags: ["all"] }, () => {
       cy.clearSessionData().loginUserViaRequest();
     });
 
-    it("should create category", () => {
+    it("As an admin I should be able to create category", () => {
```

Comment:
the `it` is supposed to be treated like a prefix to the sentence, for example:
```
it("creates a new category")
it("creates category as subcategory")
it("adds product to category")
it("removes category")
...
)
```
cc @eCzerniak